app.js

```diff
@@ -1,17 +1,29 @@
 const express = require('express');
 const app = express();
 const bodyParser = require('body-parser');
-const routes = require('./routes/routes');
```

Comment:
This file is a requirement in order to access all the routes created in it. Please add this back.